presto-geospatial-toolkit/src/test/java/com/facebook/presto/geospatial/BenchmarkJtsGeometrySerde.java

```diff
@@ -0,0 +1,336 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.geospatial;
+
+import com.google.common.base.Joiner;
+import com.google.common.io.Resources;
+import io.airlift.slice.Slice;
+import org.locationtech.jts.geom.Geometry;
+import org.locationtech.jts.io.ParseException;
+import org.locationtech.jts.io.WKTReader;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Fork;
+import org.openjdk.jmh.annotations.Measurement;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.Warmup;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+import org.openjdk.jmh.runner.options.VerboseMode;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import static com.facebook.presto.geospatial.BenchmarkData.GEOMETRYCOLLECTION;
+import static com.facebook.presto.geospatial.BenchmarkData.LINESTRING;
+import static com.facebook.presto.geospatial.BenchmarkData.MULTILINESTRING;
+import static com.facebook.presto.geospatial.BenchmarkData.MULTIPOINT;
+import static com.facebook.presto.geospatial.BenchmarkData.MULTIPOLYGON;
+import static com.facebook.presto.geospatial.BenchmarkData.POINT;
+import static com.facebook.presto.geospatial.BenchmarkData.POLYGON;
+import static com.facebook.presto.geospatial.JtsGeometryUtils.deserialize;
+import static com.facebook.presto.geospatial.JtsGeometryUtils.serialize;
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.Objects.requireNonNull;
+
+@State(Scope.Thread)
+@Fork(2)
+@Warmup(iterations = 3, time = 3)
```

Comment:
Adding a `timeUnit` to `Warmup` and `Measurement` will make them more readable. Also we can increase the number of iterations (we usually use 10).

Reply:
This comment also applies to `BenchmarkGeometrySerde`.

Reply:
With this configuration benchmark results are stable enough. Deviation is `~1%-2%`. I could've better of course, but still it shows the general picture. With current setting whole benchmark takes `~40m`. If there are some doubts if the result is statistically significant - we can always increase those values and run some particular benchmarks.